Worker/worker.hocon

```diff
@@ -27,5 +27,6 @@ akka {
                 down-all-if-less-than-nodes = 1
             }
         }
+		role.worker.min-nr-of-members = 1
```

Comment:
This ensures that the cluster can't mark any members as `Up` until at least 1 node of role `worker` successfully joins the other nodes who've also tried to join the cluster.